assets/scripts/ui/__tests__/KeyboardKey.test.js

```diff
@@ -1,25 +1,38 @@
 /* eslint-env jest */
 import React from 'react'
-import { shallow } from 'enzyme'
 import KeyboardKey from '../KeyboardKey'
+import { renderWithReduxAndIntl } from '../../../../test/helpers/render'
 
 describe('KeyboardKey', () => {
   it('renders a <kbd> element with string child', () => {
-    const wrapper = shallow(<KeyboardKey>foo</KeyboardKey>)
-    expect(wrapper.props().title).toBeUndefined()
-    expect(wrapper.debug()).toMatchSnapshot()
+    const wrapper = renderWithReduxAndIntl(
+      <KeyboardKey title="fo">foo</KeyboardKey>
+    )
+    expect(wrapper.getByText('foo').title).toBe('')
```

Comment:
🤔 I'm not sure why this is passing, shouldn't the `title` be `fo` based on the declaration above?

Reply:
Check the implementation of [`<KeyboardKey />`](https://github.com/streetmix/streetmix/blob/master/assets/scripts/ui/KeyboardKey.jsx#L23) - there shouldn't be a title. I think what's confusing is that this test adds a `title` prop and it doesn't need to be there.